examples/nidmm_measurement/tests/unit/test_measurement.py

```diff
@@ -0,0 +1,45 @@
+from unittest.mock import Mock
+
+import measurement
+import nidmm
+import pytest
+
+from ni_measurementlink_service.session_management import (
+    INSTRUMENT_TYPE_NI_DMM,
+    ChannelMapping,
+    PinMapContext,
+    SessionInformation,
+)
+
+
+def test___measurement_logic___measure___configures_and_reads_dmm(
+    measurement_context: Mock,
+    single_session_reservation: Mock,
+    nidmm_session: Mock,
+) -> None:
+    measurement_context.pin_map_context = PinMapContext("mypinmap", sites=[0])
+    single_session_reservation.session_info = SessionInformation(
+        "MySession",
+        "DMM1",
+        "0",
+        INSTRUMENT_TYPE_NI_DMM,
+        False,
+        [ChannelMapping("Pin1", site=0, channel="0")],
+    )
+    nidmm_session.read.return_value = 1.2345
+    nidmm_session.resolution_absolute = 100e-6
+
+    measured_value, signal_out_of_range, absolute_resolution = measurement.measure(
```

Comment:
Hi @bkeryan May I ask what the purpose of line39 and 42 is? The measured results are already returned by line 32. Does that mean to close the measurement?